### DISCUSSION TITLE
I dont see the Monthly Depreciation and Diff in the deprecation report

### DISCUSSION THREAD

**Discussion by nvelumani** [snipe/snipe-it#17574] [Q&A]
I have added the purchase cost I have created a deprecation in the setting, am I missing something ?? why am I not seeing the Monthly Depreciation and Diff ??

![image](https://user-images.githubusercontent.com/39930807/126220859-c7d70b44-8aea-4919-8121-f46446380a2f.png)

**Server (please complete the following information):**
 - Snipe-IT Version v5.1.5 build 6055 (gd9d5b4d73)
 - OS:  CentOS
 - Web Server:  Apache
 - PHP Version 7.4.19

**Desktop (please complete the following information):**
 - OS: Windows 
 - Browser chrome
 - Version 91.0.4472.124 (Official Build) (64-bit)

**Comment by welcome** [comment]
👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
